db/migrate/20190116182055_add_discovery_tables.rb

```diff
@@ -8,80 +8,11 @@ class AddDiscoveryTables < ActiveRecord::Migration
   def up
     if Gitlab::Database.postgresql?
       execute <<-EOF
```

Comment:
super minor can you update `EOF` to `SQL`, using SQL gives use syntax highlighting in vscode